transformations/factive_verb_transformation/test.json

```diff
@@ -0,0 +1,54 @@
+{
+  "type": "factive_verb_transformation",
+  "test_cases": [
+    {
+      "class": "FactiveVerbTransformation",
+      "inputs": {
+        "sentence": "He killed a street dog yesterday."
+      },
+      "outputs": [{
+        "sentence": "He confirmed that, he killed a street dog yesterday."
+      }]
+    }, {
+      "class": "FactiveVerbTransformation",
+      "inputs": {
+        "sentence": "An actress made her debut in hollywood."
+      },
+      "outputs": [{
+        "sentence": "An actress confirmed that, she made her debut in hollywood."
+      }]
+    }, {
+      "class": "FactiveVerbTransformation",
+      "inputs": {
+        "sentence": "John Watson was enjoying the summer in Baker street."
+      },
+      "outputs": [{
+        "sentence": "John Watson confirmed that, he was enjoying the summer in Baker street."
```

Comment:
These examples look great to me and an important transformation! You might want to randomize or maybe generate multiple of them. Most of the ones which I see use only the confirm verb.